### DISCUSSION TITLE
OCP deployment on vSphere

### DISCUSSION THREAD

**Discussion by medsaidbenguirat** [okd-project/okd#1185] [Q&A]
Hello,
i was asking how to begin with openshift i need to install the openshift container platform 4.6 on VMware vSphere 7.0.3 but each tutorial or documentation i follow have an closed end which doesn't help me at all so i hope that you help me to begin with the installation also i need to understand the master/worker explication because i wasn't sure if it is mandatory or it is just a simple virtual machine
thanks.
![vSphere](https://user-images.githubusercontent.com/103568896/163146958-e1db3692-6540-43f2-a271-ea0f4506cdf4.png)

**Comment by fortinj66** [comment]
For OKD: https://docs.okd.io/4.10/installing/installing_vsphere/preparing-to-install-on-vsphere.html
